### PR TITLE
add stress test for stream creation

### PR DESCRIPTION
**Pull request by scchan**

### CODE REVIEW

File: tests/src/stress/hipStressStreamCreate.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/*
+Copyright (c) 2015-2019 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (emankov) [inline tests/src/stress/hipStressStreamCreate.cpp]: Copyright (c) 2015-present

File: tests/src/stress/hipStressStreamCreate.cpp [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  bool descending = true;
+  for (auto i = 0; i < num_threads; ++i) {
+    threads.push_back(std::thread(run, i, descending, num_streams_per_priority));
+    descending = !descending;
+  }
+  for (auto& t : threads) {
+    t.join();
+  }
+  passed();
+}
```
Comment (emankov) [inline tests/src/stress/hipStressStreamCreate.cpp]: newline at the end of file

File: tests/src/stress/hipStressStreamCreate.cpp [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    HIPCHECK(hipStreamSynchronize(r.s)); 
+  }
+}
+
+int main(int argc, char* argv[]) {
+
+  int num_threads = 8;
+  int num_streams_per_priority = 24;
+
+#ifdef _GNU_SOURCE
```
Comment (emankov) [inline tests/src/stress/hipStressStreamCreate.cpp]: Don't we want the same on windows? Maybe clara?

**Comment by mangupta** [comment]
Closing all existing pull requests that do not target the develop branch. In case this pull request is still valid, please rebase your changes against develop branch and resubmit.

**Pull request closed without merging**